crates/rpc/src/config.rs

```diff
@@ -3,32 +3,29 @@ use std::fmt::{Display, Formatter};
 use miden_node_utils::config::Endpoint;
 use serde::{Deserialize, Serialize};
 
-pub const CONFIG_FILENAME: &str = "miden-faucet.toml";
-
-// Faucet config
+// Main config
 // ================================================================================================
 
 #[derive(Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug, Serialize, Deserialize)]
-pub struct FaucetConfig {
-    /// Endpoint of the faucet
+pub struct RpcConfig {
     pub endpoint: Endpoint,
-    /// rpc gRPC endpoint in the format `http://<host>[:<port>]`.
-    pub rpc_url: String,
-    /// Location to store database files
-    pub database_filepath: String,
+    /// Store gRPC endpoint in the format `http://<host>[:<port>]`.
+    pub store_url: String,
+    /// Block producer gRPC endpoint in the format `http://<host>[:<port>]`.
+    pub block_producer_url: String,
 }
 
-impl FaucetConfig {
+impl RpcConfig {
     pub fn as_url(&self) -> String {
```

Comment:
nit: maybe a better name would be `endpoint_url` (rather than `as_url`) since we are not converting the entire struct to a single URL string? This would be applicable to other config structs as well.

Reply:
Fixed in this pull request: #332